.github/workflows/build.yml

```diff
@@ -0,0 +1,35 @@
+name: build
```

Comment:
I don't know the GitHub Actions syntax but if it works for you, it works for me.

Reply:
You can see the build result here: https://github.com/jimporter/FNPRMS/actions/runs/53972970